st2common/st2common/content/loader.py

```diff
@@ -15,22 +15,59 @@
 
 import os
 
-import six
 from yaml.parser import ParserError
+import six
 
 from st2common import log as logging
 from st2common.constants.meta import ALLOWED_EXTS
 from st2common.constants.meta import PARSER_FUNCS
 from st2common.constants.pack import MANIFEST_FILE_NAME
+from st2common.constants.runners import MANIFEST_FILE_NAME as RUNNER_MANIFEST_FILE_NAME
 
 __all__ = [
+    'ContentRunnerLoader',
     'ContentPackLoader',
     'MetaLoader'
 ]
 
 LOG = logging.getLogger(__name__)
 
 
+class ContentRunnerLoader(object):
```

Comment:
Maybe just RunnersLoader / RunnerMetadataLoader or similar is a better name? "ContentRunner" makes it sound a bit confusing, at least to me.

With prefix pack loader with content because we used to call them "content packs" (these days days we mostly just call them packs or integration packs).